plugins/usbdmx/Siudi.cpp

```diff
@@ -0,0 +1,155 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Siudi.cpp
+ * The synchronous SIUDI widgets.
+ * Copyright (C) 2023 Alexander Simon
+ */
+
+#include "plugins/usbdmx/Siudi.h"
+
+#include <string.h>
+#include <unistd.h>
+
+#include "libs/usb/LibUsbAdaptor.h"
+#include "ola/Constants.h"
+#include "ola/Logging.h"
+#include "plugins/usbdmx/AsyncUsbSender.h"
+#include "plugins/usbdmx/ThreadedUsbSender.h"
+
+namespace ola {
+namespace plugin {
+namespace usbdmx {
+
+using ola::usb::LibUsbAdaptor;
+
+namespace {
+
+static const uint8_t ENDPOINT = 2;
+static const unsigned int TIMEOUT = 50; // 50ms is ok
+enum {SIUDI_PACKET_SIZE = 512};
```

Comment:
This could be assigned to be our DMX frame size constant, or might even become redundant as per later comments.

Reply:
Now using `ola::DMX_UNIVERSE_SIZE` in 96e10f70e7d02ce70151973234a2cd4a310aeebf